webpack.config.js

```diff
@@ -160,7 +161,10 @@ if ( calypsoEnv === 'desktop' ) {
 
 const jsLoader = {
 	test: /\.jsx?$/,
-	exclude: /node_modules/,
+	include: [
+		path.resolve( './client' ),
+		fs.realpathSync( './node_modules/notifications-panel' )
```

Comment:
I'm out of my league here. makes me nervous not to pin because of minor version updates breaking things. if we define `peerDependencies` _and_ `dependencies` will it ignore the `dependencies` when installing as a node package, vs ignoring the `peerDependencies` when building that repo separately?